src/components/App/App.js

```diff
@@ -1,26 +1,14 @@
 import Homepage from '../Homepage/Homepage'
 import Datepage from '../Datepage/Datepage'
 import NotFound from '../NotFound/NotFound'
-// import cardData from '../../cardMockData';
 import './App.css'
 import { useState, useEffect } from 'react';
 import { Routes, Route} from 'react-router-dom'
 import { createConsumer } from '@rails/actioncable';
 
-
-
 const App = () => {
   const [wins, setWins] = useState([])
   const [date, setDate] = useState(null)
-  // need to update to setDate
-  // make sure they cannot select future date
-
-  useEffect(() => {
-    fetch('http://localhost:3000/api/v1/wins')
-      .then(res => res.json())
-      .then(data => setWins(data.data))
-      .catch(err => console.log(err))
-    }, [])
```

Comment:
moved this to homepage so it would render everytime homepage component renders

Reply:
✅